draft-irtf-cfrg-aead-limits.md

```diff
@@ -396,33 +396,44 @@ v <= (p * 2^127) / (L + 1)
 ## AEAD_CHACHA20_POLY1305
 
 The known single-user analyses for AEAD_CHACHA20_POLY1305
-{{ChaCha20Poly1305-SU}}, {{ChaCha20Poly1305-MU}} combine the confidentiality and
-integrity limits into a single expression, covered below. For this AEAD, `n =
-512`, `k = 256`, and `t = 128`; the length `L` is the sum of AAD and plaintext
-(in blocks of 128 bits), see {{ChaCha20Poly1305-MU}}.
-
+{{ChaCha20Poly1305-SU}}, {{ChaCha20Poly1305-MU}} give a combined AE limit,
+which we separate into confidentiality and integrity limits below. For this
+AEAD, `n = 512`, `k = 256`, and `t = 128`; the length `L` is the sum of AAD
+and plaintext (in blocks of 128 bits), see {{ChaCha20Poly1305-MU}}.
 
 <!--
     In {{ChaCha20Poly1305-SU}}, L is |AAD| + |plaintext| + 1; the + 1 is one
     block length encoding.
 
-    From {{ChaCha20Poly1305-MU}} Theorem 4.1 / 3.4:
-      AE <= v * 2^25 * (L+1) / 2^t
-    where t = 128.
-    (NB: The bound component "c * L" (for c = 3*2^24) is upper-bounding
-    2^25 * (L+1) for the worst case L = |AAD|+|m| = 2; cf. Theorem 3.4.)
+    From {{ChaCha20Poly1305-MU}} Theorem 4.1:
+      AEA <= PRF-advantage  +  v * 2^25 * (L+1) / 2^t
+    where t = 128. The CA part of this is only the PRF advantage, as in the
+    proof of Theorem 4.1, the hops bounding G_3 only apply to the decryption
+    oracle. So CA beyond the PRF advantage is 0.
 -->
+
+### Confidentiality Limit
+
 ~~~
-AEA <= (v * (L + 1)) / 2^103
+CA <= 0
 ~~~
 
-This advantage is a tight reduction based on the underlying Poly1305 PRF {{!Poly1305=DOI.10.1007/11502760_3}}.
-It implies the following limit:
+This implies there is no limit beyond the PRF security of the underlying ChaCha20
```

Comment:
Should we say "assumes" instead of "implies"?  I know that this is a dirty word here, but:

```suggestion
This analysis assumes that there is no limit beyond the PRF security of the underlying ChaCha20
```

Reply:
No, the analysis indeed _implies_ that there is no limit beyond the _assumed_ PRF security of ChaCha20.  (The latter is in line with our statements e.g. under Security Considerations, that analyses assume primitives ideal, like PRF being truly random functions.)

Fine to add "_assumed_ PRF security ..." though, if that makes it clearer.